demo/example.html

```diff
@@ -1,35 +1,50 @@
-            null, // divider
-            {label:'Sheep',         icon:'icons/application-monitor.png',         action:function() { alert('clicked 4') } },
-            {label:'Cheese',        icon:'icons/bin-metal.png',                   action:function() { alert('clicked 5') } },
-            {label:'Bacon',         icon:'icons/magnifier-zoom-actual-equal.png', action:function() { alert('clicked 6') } },
-            null, // divider
-            {label:'Onwards',       icon:'icons/application-table.png',           action:function() { alert('clicked 7') } },
-            {label:'Flutters',      icon:'icons/cassette.png',                    action:function() { alert('clicked 8') } }
-          ]
-        });
-      });
-    </script>
+
   </head>
   <body style="background-color:#000000; color:#ffffff; font-family:arial,sans-serif">
 
-    <div id="mythingy" style="width:500px; height: 500px; margin: 10px; border:2px dashed #666666; padding:6px">
+    <div id="mythingy" style="width:500px; height: 500px; margin: 10px; border:2px dashed #666666; padding:6px" data-sometoken="importantReferenceOnOriginalElement" data-anothertoken="anotherReference">
       right click in this box to show custom context menu 
     </div>
 
     right click out of the box to show the standard browser menu (if you're trying to view-source, right click here)
+
+
+    <script src="http://code.jquery.com/jquery-1.11.2.min.js"></script>
+    <script src="../jquery.contextmenu.js"></script>
+    <script>
+        $(function() {
+            $('#mythingy').contextPopup({
+                title: 'My Popup Menu',
+                items: [
+                    {label:'Some Item',     icon:'icons/shopping-basket.png',             action:function() { alert('clicked 1') } },
+                    {label:'Another Thing', icon:'icons/receipt-text.png',                action:function() { alert('clicked 2') } },
+                    {label:'Blah Blah',     icon:'icons/book-open-list.png',              action:function() { alert('clicked 3') } },
+                    null, // divider
+                    {label:'Sheep',         icon:'icons/application-monitor.png',         action:function() { alert('clicked 4') } },
+                    {label:'Cheese',        icon:'icons/bin-metal.png',                   action:function() { alert('clicked 5') } },
+                    {label:'Bacon',         icon:'icons/magnifier-zoom-actual-equal.png', action:function() { alert('clicked 6') } },
+                    null, // divider
+                    {label:'Onwards',       icon:'icons/application-table.png',           action:function() { alert('clicked 7') } },
+                    {label:'Flutters',      icon:'icons/cassette.png',                    action:function() { alert('clicked 8') } }
+                ],
+                dataTokenLookups: [
+                    'data-sometoken',
```

Comment:
Are these tokens meant to be associated with the items? If so, it would be much easier to understand for end users and less prone to developer error if they were in the items array. e.g. {label:'...', icon:'...', dataTokenLookup:'...'}